css/selectors/matches.json

```diff
@@ -8,60 +8,95 @@
           "support": {
             "chrome": [
               {
-                "version_added": "66"
+                "version_added": "66",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#enable-experimental-web-platform-features",
+                    "value_to_set": "enabled"
+                  }
+                ],
+                "notes": [
+                  "It has issues with combinators (<a href='https://crbug.com/842157'>Chromium bug 842157</a>)"
+                ]
               },
               {
                 "version_added": "12",
-                "alternative_name": ":-webkit-any"
+                "alternative_name": ":-webkit-any",
+                "notes": [
+                  "It does not support combinators."
+                ]
               }
             ],
             "chrome_android": [
               {
-                "version_added": "66"
+                "version_added": "66",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#enable-experimental-web-platform-features",
+                    "value_to_set": "enabled"
+                  }
+                ],
+                "notes": [
+                  "It has issues with combinators (<a href='https://crbug.com/842157'>Chromium bug 842157</a>)"
+                ]
               },
               {
                 "version_added": "18",
-                "alternative_name": ":-webkit-any"
-              }
-            ],
-            "edge": [
-              {
-                "version_added": "15"
-              },
-              {
-                "version_added": "12",
-                "alternative_name": ":-ms-matches"
+                "alternative_name": ":-webkit-any",
+                "notes": [
+                  "It does not support combinators."
+                ]
               }
             ],
+            "edge": {
+              "version_added": false
+            },
             "edge_mobile": {
               "version_added": false
             },
             "firefox": {
               "version_added": "4",
-              "alternative_name": ":-moz-any"
+              "alternative_name": ":-moz-any",
+              "notes": [
+                "It does not support combinators."
+              ]
             },
             "firefox_android": {
               "version_added": "4",
-              "alternative_name": ":-moz-any"
+              "alternative_name": ":-moz-any",
+              "notes": [
+                "It does not support combinators."
+              ]
             },
             "ie": {
               "version_added": false
             },
             "opera": {
               "version_added": true,
-              "alternative_name": ":-webkit-any"
+              "alternative_name": ":-webkit-any",
```

Comment:
This is also behind a flag: https://www.chromestatus.com/features/5445716612743168